deploy/helm/observability/loki/values.yaml

```diff
@@ -0,0 +1,219 @@
+    accessKeySecret: "minio123"
+    bucket: loki
+    endpoint: http://minio-observability-storage.observability-hub.svc.cluster.local:9000
+
+# RBAC configuration
+rbac:
+  create: true
+
+  # Collector service account and RBAC setup
+  collector:
+    # Set to true to create collector SA and RBAC (required for new deployments)
+    # Set to false if collector SA already exists
+    create: true
+    serviceAccount:
+      name: collector
+      namespace: openshift-logging
+
+  # Service accounts for log collection
+  serviceAccounts:
+    logForwarder:
+      name: loki-log-forwarder
+      create: true
+
+  # Cluster roles for tenant access
+  clusterRoles:
+    tenantLogs:
+      create: true
+
+# Security configuration
+podSecurityContext:
+  runAsNonRoot: true
+
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+
+# Node selector
+nodeSelector: {}
+
+# Tolerations
+tolerations: []
+
+# Affinity
+affinity: {}
+
+# Additional labels and annotations
+commonLabels: {}
+commonAnnotations: {}
+
+# Cluster logging configuration
+clusterLogging:
+  enabled: true
+
+  # ClusterLogForwarder configuration
+  logForwarder:
+    enabled: true
+
+    # Service account for log collection
+    serviceAccount:
+      name: collector
+      create: false # Use existing service account from OpenShift Logging
+
+    # Target LokiStack configuration
+    target:
+      name: logging-loki
+      namespace: observability-hub
+
+    # TLS configuration for secure communication
+    tls:
+      # CURRENT WORKING CONFIG: Uses insecureSkipVerify for internal cluster communication
+      # Set to false and configure CA for production security
+      insecureSkipVerify: true
+      # CA configuration for validating Loki gateway certificates (when insecureSkipVerify: false)
+      ca:
+        configMapName: openshift-service-ca.crt
+        key: service-ca.crt
+
+    # Input configuration for log filtering
+    inputs:
+      application:
+        # Collect from all namespaces (no filtering)
+        allNamespaces: true
+        customName: application-all-namespaces
+      infrastructure:
+        # Filter infrastructure logs to reduce volume
+        filtered: true
+        customName: infrastructure-minimal
+        sources:
+          - node # Node-level infrastructure logs
+          - container # Container runtime logs
+      audit:
+        # Audit logs disabled due to high volume (can be re-enabled)
+        enabled: false
+
+    # Batch configuration for log forwarding
+    batch:
+      maxSize: 1MB
+      maxRecords: 1000
+      timeout: 5s
+
+    # Rate limiting per tenant
+    rateLimiting:
+      enabled: true
+      audit:
+        maxLogsPerSecond: 1000
+      application:
+        maxLogsPerSecond: 500
+      infrastructure:
+        maxLogsPerSecond: 500
+
+# MCP Server configuration for log querying
+mcpServer:
+  enabled: true
```

Comment:
where will be these settings used ?
```
mcpServer:
  enabled: true
```

Reply:
Maybe I'll just actually remove the mcp configuration stuff since we've settled on korrel8, I don't have to add this anymore.

Reply:
I've modified this for general external access for when we implement korrel8